### ISSUE TITLE
Option for Checked-box to be higher contrast or different color?

### ISSUE THREAD

**Issue by JamesO2x** [mjbvz/vscode-markdown-checkboxes#13]
The current default box when checked is "grey on grey" and incredibly hard to see at a distance. Is there any way to make the box turn Green when checked? Or even just use a higher contrast "black on white" color scheme? The current look is basically unreadable, thus effectively useless at visually communicating what its supposed to communicate.

![Imgur](https://i.imgur.com/EUWYspP.png)

The specifics of the color/design aren't important here. There just needs to be _something_ with a little more contrast.

Where exactly is this extension pulling its checkbox graphics from? Is it using something from VS Code's currently installed theme, or are the checkbox graphics custom? Or are the boxes generated with CSS or something?

**Comment by mjbvz** [comment]
What OS and theme are you using? Checked checkboxes show up as blue for me with the default VS Code theme on MacOS

<img width="354" alt="Screen Shot 2022-03-07 at 5 15 11 PM" src="https://user-images.githubusercontent.com/12821956/157146230-50363c66-00aa-47e3-b1e5-7fcc27841e33.png">

**Comment by JamesO2x** [comment]
Oh weird. Then it must be a theme issue and not anything to do with this extension specifically.

I'm running standard Windows 10, and using the popular "Dracula" theme.

I tried changing my Color Theme, and now I'm getting all sorts of weird color inconsistencies in the editor. I may have messed something up in my `settings.json` file a while ago, but never noticed because everything more or less looked fine.

I'm gonna try clearing out my user settings and see what happens.

**Comment by JamesO2x** [comment]
Update: I found out it was a conflict issue with the "Mark Down All in One" extension. Apparently that extension also supports Checkboxes... but when I first installed it the checkboxes were not working and I assumed it didn't support them. Which is why I ended up installing this Markdown checkboxes extension.

I disabled both extensions, then enabled them one-at-a-time, then both again. So it seems _this_ Extension (Markdown Checkboxes) is working just fine (I get blue checkmarks now). But its the _other_ extension that is working oddly. After restarting VS Code a couple times, I got "Mark Down All in One" to render checkboxes now, and it it seems _that_ extension is the one that is causing checkboxes to be all grey.

Anyway, so my issue seems to be with that extension. Sorry for the confusion!
